tests/lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/prefer-ember-test-helpers');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const TEST_FILE_NAME = 'some-test.js';
+
+ruleTester.run('prefer-ember-test-helpers', rule, {
+  valid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { blur } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await blur();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { find } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await find();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { focus } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await focus();
+      });`,
+    },
+  ],
+
+  invalid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: `test('foo', async (assert) => {
+        await blur();
+      });`,
+      output: null,
+      errors: [
+        {
+          message: 'Import the `blur()` method from @ember/test-helpers',
+        },
+      ],
+    },
+    {
+      filename: TEST_FILE_NAME,
+      code: `test('foo', async (assert) => {
+        await find();
+      });`,
+      output: null,
+      errors: [
+        {
+          message: 'Import the `find()` method from @ember/test-helpers',
+        },
+      ],
+    },
+    {
+      filename: TEST_FILE_NAME,
+      code: `test('foo', async (assert) => {
+        await focus();
+      });`,
+      output: null,
+      errors: [
+        {
+          message: 'Import the `focus()` method from @ember/test-helpers',
+        },
+      ],
+    },
+  ],
```

Comment:
Let's add an invalid test case to ensure we ignore `focus` imported from wrong place.

```
import { focus } from 'irrelevant-import-path';
...
focus()
```

Reply:
Wait, to confirm - is this a valid case or an invalid case? (like, should throw an error or not?) :)

Reply:
Ah sorry, that's a typo, should be valid test case:

```js
{
  filename: TEST_FILE_NAME,
  code: `import { focus } from 'irrelevant-import-path';
         focus()`,
},
```

If someone wants to use their own `focus` function from a different source or that they defined on their own, we have no problem with them doing that. So we need this valid test case too:

```js
{
  filename: TEST_FILE_NAME,
  code: `function focus() { console.log('focusing!'); }
         focus()`,
},
```